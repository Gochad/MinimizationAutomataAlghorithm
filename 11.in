q0 q1 q2 q3 q4 q5 q6 q7
q0
q2
0 1
q0 0 q1
q0 1 q5
q1 0 q6
q1 1 q2
q2 0 q0
q2 1 q2
q3 0 q2
q3 1 q6
q4 0 q7
q4 1 q5
q5 0 q2
q5 1 q6
q6 0 q6
q6 1 q4
q7 0 q6
q7 1 q2