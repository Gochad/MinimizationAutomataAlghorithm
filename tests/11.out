q6 q0q4 q1q7 q3q5 q2 
q0
q2 
0 1 
q6 0 q6
q6 1 q0q4
q0q4 0 q1q7
q0q4 1 q3q5
q1q7 0 q6
q1q7 1 q2
q3q5 0 q2
q3q5 1 q6
q2 0 q0q4
q2 1 q2
